### DISCUSSION TITLE
Welcome to hyprland-per-window-layout Discussions!

### DISCUSSION THREAD

**Discussion by coffebar** [coffebar/hyprland-per-window-layout#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ProMix0** [comment]
I want to thank you. That plugin is magnificent! I has needed this feature since I was left Win7. No one WM or DE what I know doesn't support this. And Your plugin - Your work - is a gorgeous Gods gift. Really, really thank You!

**Reply by coffebar** [reply]
Thanks so much) :heart: 
Actually, you have many options to get this functionality on Xorg. I believe XFCE has it out of the box. Previously I used [kbdd](https://github.com/qnikst/kbdd). But it doesn't support Wayland.
